example/Thewissen.PancakeViewSample.Android/Thewissen.PancakeViewSample.Android.csproj

```diff
@@ -72,6 +75,9 @@
     <PackageReference Include="Xamarin.Android.Support.Media.Compat">
       <Version>28.0.0.1</Version>
     </PackageReference>
+    <PackageReference Include="Xamarin.Forms.PancakeView">
+      <Version>1.2.0</Version>
```

Comment:
There's no such thing as a 1.2.0 PancakeView sir 😂

Reply:
yup indeed :) I created the nuget 1.2.0 locally so I could test the modifications in the sample app, since I couldn't reference the projects directly cause of this nice multi targeting trick :)

Reply:
Yeah. Its a PITA that that isn't on Mac yet. Have to jump through a lot of loops to get it to work, but I just really like how simple it keeps things.